### PR TITLE
feat: [M3-6941]- Uniform Spacing b/w Resource Link Columns In Empty State Landing Pages

### PR DESCRIPTION
**Pull request by carrillo-erik**
## Description 📝
This PR addresses the issue with a lack of uniform spacing between the **Getting Started Guides** and **Video Playlist** sections on the empty state landing pages.

## Changes  🔄
List any change relevant to the reviewer.
- Updated the `width` of the `ResourcesLinkSection` by adding a breakpoint declaration for `mid`.
- Replaced default export for `KubernetesLanding` with named export.

## Preview 📷
| Before  | After   |
| ------- | ------- |
| <video src="https://github.com/user-attachments/assets/7c37e1bf-b986-4ba2-8181-7ffc138356ed" /> | <video src="https://github.com/user-attachments/assets/b274d863-6f9a-4dcd-9cca-0c152f23cd26" /> |

### Prerequisites
(How to setup test environment)
- Have a user account without any compute instances in order to display the empty state landing pages for each entity.

### Verification steps
(How to verify changes)
- Compare the spacing between the **Getting Started Guides** and **Video Playlist** sections on the empty state landing pages in the develop branch against this PR.
- Verify that the spacing is now uniform across entities with both **Getting Started Guides** and **Video Playlist** sections.
  - Refer to the videos in the **Preview** section to verify changes.

## As an Author I have considered 🤔
- [x] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
- [ ] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [x] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

File: packages/manager/src/features/PlacementGroups/PlacementGroupsLanding/PlacementGroupsLandingEmptyState.tsx [hunk 1]
```diff
@@ -39,6 +39,7 @@ export const PlacementGroupsLandingEmptyState = ({
       headers={headers}
       icon={LinodeIcon}
       linkAnalyticsEvent={linkAnalyticsEvent}
+      wide
```
Comment (carrillo-erik) [inline packages/manager/src/features/PlacementGroups/PlacementGroupsLanding/PlacementGroupsLandingEmptyState.tsx:42]: This allows the resource section to be centered.

File: packages/manager/src/features/VPCs/VPCLanding/VPCEmptyState.tsx [hunk 2]
```diff
@@ -42,6 +42,7 @@ export const VPCEmptyState = () => {
       headers={headers}
       icon={NodeBalancerIcon}
       linkAnalyticsEvent={linkAnalyticsEvent}
+      wide
```
Comment (carrillo-erik) [inline packages/manager/src/features/VPCs/VPCLanding/VPCEmptyState.tsx:45]: This allows the resource section to be centered.

File: packages/manager/src/components/Placeholder/Placeholder.tsx
Comment (carrillo-erik) [inline packages/manager/src/components/Placeholder/Placeholder.tsx:1]: These are just lint changes.

File: packages/manager/src/features/VPCs/VPCLanding/VPCEmptyState.tsx (hunk 2 above)
Comment (dwiley-akamai) [reply packages/manager/src/features/VPCs/VPCLanding/VPCEmptyState.tsx:45]: I wasn't seeing a difference for this and Placement Groups on this branch vs. off of it, seemed like it was already centered regardless of breakpoint?

**Review by dwiley-akamai — APPROVED** [review]

**Review by harsh-akamai — APPROVED** [review]
Changes look good🎉  Thanks @carrillo-erik !

File: packages/manager/src/features/VPCs/VPCLanding/VPCEmptyState.tsx (hunk 2 above)
Comment (carrillo-erik) [reply packages/manager/src/features/VPCs/VPCLanding/VPCEmptyState.tsx:45]: Sorry, the comment meant to read "This allows the resource section to be centered in *mobile view*."

**Pull request merged**